test/common/ssl/context_impl_test.cc

```diff
@@ -363,35 +394,146 @@ TEST(ClientContextConfigImplTest, InvalidCertificateHash) {
 // TODO(PiotrSikora): Support multiple TLS certificates.
 TEST(ClientContextConfigImplTest, MultipleTlsCertificates) {
   envoy::api::v2::auth::UpstreamTlsContext tls_context;
+  Secret::MockSecretManager secret_manager;
   tls_context.mutable_common_tls_context()->add_tls_certificates();
   tls_context.mutable_common_tls_context()->add_tls_certificates();
-  EXPECT_THROW_WITH_MESSAGE(ClientContextConfigImpl client_context_config(tls_context),
-                            EnvoyException,
-                            "Multiple TLS certificates are not supported for client contexts");
+  EXPECT_THROW_WITH_MESSAGE(
+      ClientContextConfigImpl client_context_config(tls_context, secret_manager), EnvoyException,
+      "Multiple TLS certificates are not supported for client contexts");
+}
+
+TEST(ClientContextConfigImplTest, StaticTlsCertificates) {
+  std::string kExpectedCertificateChain =
+      R"EOF(-----BEGIN CERTIFICATE-----
```

Comment:
are these two strings the same as ones in secret_manager_impl_test.cc?  If so, can they be defined once?

Reply:
Created and used separated test data library.